pyk/src/pyk/cterm/symbolic.py

```diff
@@ -94,6 +96,28 @@ def kast_to_kore(self, kinner: KInner) -> Pattern:
     def kore_to_kast(self, pattern: Pattern) -> KInner:
         return kore_to_kast(self._definition, pattern)
 
+    def minimize_constraints(self, constraints: tuple[KInner, ...], path_condition: KInner) -> tuple[KInner, ...]:
+        """Minimize given branching constraints with respect to a given path condition."""
+        # By construction, this function is to be called with at least two sets of constraints
+        assert len(constraints) >= 2
+        # Determine intersection between all returned sets of branching constraints
+        flattened_default = tuple(flatten_label('#And', c) for c in constraints)
```

Comment:
Nitpick:

```suggestion
        flattened_default = [flatten_label('#And', c) for c in constraints]
```

(This reads nicer, and `flattened_default` does not need to be immutable.)

Reply:
Yes, done.